frontend/lib/main.dart

```diff
@@ -23,29 +23,29 @@ class MyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) => MaterialApp(
-    title: 'APDV Demo',
-    theme: ThemeData(
-      primarySwatch: Colors.blue,
-    ),
-    home: FutureBuilder(
-      future: userGateway.isMemoryTokenValid(),
-      builder: (context, snapshot) {
-        if (!snapshot.hasData) {
-          return loadingInCenter();
-        }
-        if (snapshot.data == true) {
-          return App(
-            userGateway: userGateway,
-            endpointGateway: EndpointGateway(userGateway),
-          );
-        } else {
-          return LoginView(
-            userGateway: userGateway,
-          );
-        }
-      },
-    ),
-  );
+        title: 'APDV Demo',
```

Comment:
Mam wrażenie, że z @zbrze macie różne style ustawione, bo pojawiają się takie różnice w PR'kach. Fajnie jakbyście to ze sobą wspólnie ustalili

Reply:
Nwm jaką decyzję podjeliście z @zbrze, ale uważam, że też można na to formatowanie zerknąć, czemu takie różnice występują 